src/mlpack/methods/ann/rbm/rbm_impl.hpp

```diff
@@ -50,6 +50,7 @@ RBM<InitializationRuleType, DataType, PolicyType>::RBM(
     reset(false)
 {
   numFunctions = this->predictors.n_cols;
+  steps = 0;
```

Comment:
Do you mind to use the member initializer list for the ``steps`` parameter for consistency?